### PR TITLE
Fix two bugs in Future

### PR DESCRIPTION
**Pull request by mkoval**
- Fix a logic error in `add_done_callback`.
- Fix reference to non-existent logger in `_set_done`.

### CODE REVIEW

File: src/prpy/futures.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def add_done_callback(self, fn):
         @type  fn: (ResultType) -> None
         """
         with self.lock:
-            if self._is_done:
+            if not self._is_done:
                 if fn in self._callbacks:
```
Comment (psigen) [inline src/prpy/futures.py]: Should this check be moved outside this `if` block entirely? it seems like it applies to both cases.

**Comment by cdellin** [comment]
Wow, you guys are already fixing bugs in the future? All the bugs in the present are already solved?

**Comment by gilwoolee** [comment]
Haha yeah that's what I thought too when I first got this email.

- Gilwoo

File: src/prpy/futures.py (hunk 1 above)
Comment (mkoval) [inline src/prpy/futures.py]: It does seem that way. 👍

**Comment by gilwoolee** [comment]
Thanks @mkoval !

**Pull request merged**